scicomp/ssh.rst

```diff
@@ -64,9 +81,28 @@ While there are many options for the key generation program ``ssh-keygen``, here
 
 Here are our recommended input options for key generation.
 
-::
+.. tabs::
+
+  .. group-tab:: Windows
+    
+     ::
+      
+         ssh-keygen -t rsa -b 4096 -f %USERPROFILE%\.ssh\id_rsa_triton -C "triton key for %USERNAME%"     
```

Comment:
Normally I would recommend one key per client computer, for all hosts it would connect to.  What are your thoughts on that?

I can think of some reasons for multiple (e.g. on Triton I have one passwordless one for internal ssh and one with password for git).  And maybe being more controlled has less risk of being interefred by other things.

Also, in the comments, I would always include the source computer.  When you look at Triton and see all the old keys there, you want to be able to easily see what is still relevant and what is obsolete.  (at least for me default sometimes seems OK)